pom.xml

```diff
@@ -29,11 +29,11 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.9</version>
+    <version>2.11</version>
   </parent>
 
   <artifactId>bouncycastle-api</artifactId>
-  <version>1.648.4-SNAPSHOT</version>
+  <version>2.16.0-SNAPSHOT</version>
```

Comment:
So this is the version of core that the plugin goes with. After this point it will be decoupled.

It would have been better if this had been done back in 1.648 or in time for 2.0 but we are where we are.